crc/aarch64/crc32_gzip_refl_pmull.h

```diff
@@ -47,11 +47,17 @@
 .equ	br_high_b2, 0x1
 
 	.text
+#ifndef __MACH__
 	.section	.rodata
+#else
+  .section	__DATA,data
```

Comment:
Dose it  mean readonly data section ?

Reply:
It seems we should use __TEXT,__const according to https://developer.apple.com/library/archive/documentation/Performance/Conceptual/CodeFootprint/Articles/MachOOverview.html . Fixed.